src/Sarif/Writers/MimeType.cs

```diff
@@ -73,6 +73,8 @@ public static string DetermineFromFileExtension(Uri fileUri)
         public static readonly string Directory = "application/x-directory";
         /// <summary>The MIME type used for CSharp files.</summary>
         public static readonly string CSharp = "text/x-csharp";
+        /// <summary>The MIME type for SARIF files.</summary>
+        public static readonly string Sarif = "text/x-sarif";
```

Comment:
Fixed: Set MIME type to "application/sarif+json".

---
In reply to: [77440108](https://github.com/Microsoft/sarif-sdk/pull/501#discussion_r77440108) [](ancestors = 77440108,76704516)